src/Module.Server/ModuleData/dtv/dtv_characters.xml

```diff
@@ -2287,7 +2287,7 @@
       </EquipmentRoster>
     </Equipments>
   </NPCCharacter>
-  <NPCCharacter id="crpg_dtv_highland_lieutenant" name="{=}Chosen" race="light_bot">
+  <NPCCharacter id="crpg_dtv_highland_lieutenant" name="{=}Highland Chosen" race="light_bot">
```

Comment:
How about 'Chosen Highlander'?

Reply:
^I like that, another reference to the movie